editor/html/ketcher_iframe.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
This HTML does not look well-formed.

Reply:
Whoops, will move the script tags into body. I can add a head too.

Outside of this, would you suggest any other changes? Because this is only meant to show Ketcher, I'd prefer to keep it smaller, if possible.

Reply:
(BTW, pushed changes. It's having trouble showing up for me; just in case, [here's a link](https://github.com/Open-Reaction-Database/ord-schema/blob/editor-ketcher/editor/html/ketcher_iframe.html).)

Reply:
I have no other suggestions, but I can't say I understand why an iFrame is helpful.

The only time I expect to see an iFrame is when someone needs a separate JS context for code from a different origin. Since we are hosting the JS, I would need to know the purpose.

Reply:
iFrames are EPAM's (the organization behind Ketcher) recommended way to add Ketcher to another page. I tried to use Ketcher in a regular div, but ran into difficulties that were tough to fix and required our own changes.

We wanted to avoid our own changes as much as possible, and also to save time -- we figured it would be better to stick with an iFrame.